includes/types/CMB2_Type_Multi_Base.php

```diff
@@ -31,7 +31,7 @@ public function select_option( $args = array() ) {
 	 * @param  int   $i    Iterator value
 	 * @return string       Gnerated list item html
 	 */
-	public function list_input( $args = array(), $i ) {
+	public function list_input( $args = array(), $i = null ) {
```

Comment:
Updating to match the default value for the first argument to `CMB2_Types::_id`, which is where this `$i` argument is used.
```suggestion
	public function list_input( $args = array(), $i = '' ) {
```